core/src/actors/session/handlers.rs

```diff
@@ -77,6 +77,13 @@ impl StreamHandler<BytesMut, Error> for Session {
                     ) => {
                         peer_discovery_peers(&peers);
                     }
+                    ////////////////////
+                    // INVENTORY      //
+                    ////////////////////
+                    // Handle Inv message
+                    (SessionType::Outbound, SessionStatus::Consolidated, Command::Inv(inv)) => {
```

Comment:
If we are sending `inv`s only to outbounds, we strictly need to accept `inv`s from inbounds. Otherwise, inventory items diffusion will just not work. 
```suggestion
                    (_, SessionStatus::Consolidated, Command::Inv(inv)) => {
```